__tests__/api.test.js

```diff
@@ -19,4 +19,15 @@ describe("/api/topic", () => {
             });
         })
     })
+    describe("/api", () => {
+        test("Responds with status 200 and serves json file", () => {
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({ body: data })=>{
+                    expect(Object.keys(data).length).not.toBe(0);
+                    expect(typeof data).toBe('object');
+                           });
+            })
+        })    
```

Comment:
You can also use a little trick to test concisely here - requiring in the endpoints.json then testing that the response from the server is the same as it. This keeps our test very precise (as you're testing the entire endpoints object, instead of just that it's an object)